.coafile

```diff
@@ -12,12 +12,22 @@ use_spaces = True
 # will be executed sequentially; also we need the LineLengthBear to double
```

Comment:
Ah I didn't realize it was inactive previously. I agree, your version sounds better.